docs/03-tutorials/00-application-connectivity/ac-07-revoke-client-cert.md

```diff
@@ -7,7 +7,7 @@ You can revoke a client certificate generated for your Application. Revocation p
 To revoke a client certificate, send a request to the `certificates/revocations` endpoint. Pass the certificate you want to revoke and a key that matches this certificate in the call. Run:
     
 ```bash
-curl -X POST https://gateway.{CLUSTER_DOMAIN}/v1/applications/certificates/revocations --cert {CERT_TO_REVOKE} --key {CERT_TO_REVOKE_KEY} -k 
+curl -X POST https://gateway.{CLUSTER_DOMAIN}/v1/applications/certificates/revocations --cert {PATH_TO_CLIENT_CERT_TO_REVOKE} --key {PATH_TO_KEY} -k 
```

Comment:
This tutorial could also be changed to use environment variables instead of placeholders.

Reply:
One note concerning renewal via internal API. The tutorial doesn't mention that the code should be executed within a pod running in a cluster. We should also mention that `port-forward` could be used.

Reply:
**TOCONSIDER**/**TODO**